torch/ao/quantization/fx/convert.py

```diff
@@ -124,16 +124,26 @@ def _replace_observer_with_quantize_dequantize_node(
             graph.erase_node(node)
         return
 
-    # otherwise, we can convert the observer module call to quantize/dequantize node
+    # otherwise, we can convert the activation_post_process module call to quantize/dequantize node
+    # high level
+    # step 1: extract information for inserting q/dq node from activation_post_process
+    # e.g. quantize_op, quantization parameters (no graph change)
+    # step 2: (optional) insert choose_qparams, and update the extracted information if needed
+    # step 3: insert q/dq node
```

Comment:
would it help to reorganize the code as 

```
if condition_1:
  # do step 1..n
elif condition_2:
  # do step 1..n
  ...
```

instead of the current organization of

```
# do step 1
if condition_1:
  ...
elif condition_2:
  ...

# do step 2
if condition_1:
  ...
elif condition_2:
  ...
```

Reply:
that way the reader could just read the specific block they care about and ignore the rest when they are debugging something, versus always being forced to read all the blocks.  One could enforce correctness with tests (already exist) and code reuse as necessary, but I'd bias for simplicity and readability over maximizing reuse.  I think the addition of `is_demposed` is making the current thing too confusing

Reply:
I think we can, there might be some code duplication I think this will be clearer, can I do this after this stack is landed?